### PR TITLE
Move `NAME` and `VERSION` from ENV to ARG statement in Containerfiles

### PR DESCRIPTION
**Pull request by JayDoubleu**
Move `NAME` and `VERSION` from ENV to ARG statement in Containerfiles
Fixes: https://github.com/containers/toolbox/issues/188

### CODE REVIEW

File: images/fedora/f32/Containerfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
 FROM registry.fedoraproject.org/fedora:32
 
-ENV NAME=fedora-toolbox VERSION=32
+ENV TOOLBOX_NAME=fedora-toolbox TOOLBOX_VERSION=32
 LABEL com.github.containers.toolbox="true" \
-      com.redhat.component="$NAME" \
-      name="$NAME" \
-      version="$VERSION" \
+      com.redhat.component="$TOOLBOX_NAME" \
```
Comment (HarryMichal) [inline images/fedora/f32/Containerfile:5]: This should remain as it was. It does not reference to the name of a container but to the Toolbox/Toolbx utility.

Comment (JayDoubleu) [reply images/fedora/f32/Containerfile]: Thanks @HarryMichal, would changing it to ARGs completely be any better ?

Comment (HarryMichal) [reply images/fedora/f32/Containerfile]: Yes, I think using ARGs will be a much better option here.

**Review by debarshiray — COMMENTED** [review]
Oops, I already [merged](https://github.com/containers/toolbox/commit/9506173f88dc26bfb0229edea0d39699d636d9e2) @travier 's copy of these changes which he first published at https://github.com/containers/docs/pull/15

I should have noticed that you already had this pull request ready to go with the same changes.  I am very sorry about that.  I didn't mean to take the credit away from you.  :(

**Comment by debarshiray** [comment]
Closing.

Thanks for your contributions to Toolbx, @JayDoubleu

**Pull request closed without merging**